java/main/com/google/privacy/differentialprivacy/GaussianNoise.java

```diff
@@ -114,6 +114,32 @@ public MechanismType getMechanismType() {
     return MechanismType.GAUSSIAN;
   }
 
+  /**
+   * Returns {@link ConfidenceInterval} object with the given {@code confidenceLevel},
+   * using {@code noisedValue}, L_0 and L_inf sensitivities,
+   * and {@code epsilon} and {@code delta} for Gaussian distribution.
+   */
+  @Override
+  public ConfidenceInterval getConfidenceInterval(
```

Comment:
have we decided between get and compute?

I would call this compute since we do not simply return an interal value but actually compute something (similar to the computeResult function of Count.java )

Reply:
Initially, there was no preference between get and compute but now compute is better. I changed function names to computeConfidenceInterval.